Styling updates

xinnanlisf opened the pull request:
# Critical Changes

# Changes

# Issues Closed

# New Metadata

## Unpackaged Metadata

# Deleted Metadata

# Testing Notes

xinnanlisf commented on force-app/main/default/lwc/accountModelSettings/accountModelSettings.html:
```diff
@@ -1,123 +1,119 @@
 <template>
-    <div class="slds-grid--vertical slds-theme_default">
```
all right, github cant even figure out which line is removed...

Anyway, we don't need these css classes since inheriting from parent component.
Also, we can move slds-p-top_large slds-p-left_small to edaSettingsPage.html so that each page component wont have to add this anymore.

MatthewBlanski replied:
Have you tried rendering these on their own? We want them to be able to be inserted directly into modals.

xinnanlisf commented on force-app/main/default/lwc/edaSettings/edaSettings.html:
```diff
@@ -2,13 +2,13 @@
                 <c-eda-settings-navigation onsettingsnavigation={handleSettingsNavigation}></c-eda-settings-navigation>
             </div>
             <div
-                class="slds-col slds-no-flex slds-size_8-of-12 slds-scrollable_y eda-height_full"
```
we dont need eda-height_full here since upper level div is having it.

MatthewBlanski replied:
We do because each content area has its own definition of size. So what we're specifying in this instance is that the overall settings container needs to take up 100% of the height of the content area it's in and then within that the nav and page need to take up the height of the settings container. Then they need to be scrollable on the y axis.

xinnanlisf commented on force-app/main/default/lwc/edaSettings/edaSettings.html:
```diff
@@ -2,13 +2,13 @@
             </div>
             <div
-                class="slds-col slds-no-flex slds-size_8-of-12 slds-scrollable_y eda-height_full"
+                class="slds-col slds-no-flex slds-size_8-of-12 slds-scrollable_y slds-p-top_large slds-p-left_small"
```
adding paddings here and also move it to the end.

MatthewBlanski replied:
Adding the padding to start with here is a good idea. This should keep consistency throughout.

xinnanlisf commented on force-app/main/default/lwc/settingsSaveCanvas/settingsSaveCanvas.html:
```diff
@@ -8,7 +8,6 @@
                     label={labelReference.settingsButtonEdit}
                     title={labelReference.settingsButtonEdit}
                     onclick={handleEditClick}
-                    class="slds-p-right_x-small"
```
removing these because based on the mocks, buttons should be aligned with affordance.

xinnanlisf replied:
Done! It looks like the sizes of cancel and save are different. I left them as they were since the mocks was showing the same way.

xinnanlisf commented:
@MatthewBlanski, I made a few tweaks to styling and left comments for those changes. With these changes, individual settings page does not need to worry about styling any more also given that we will create reusable components. All they need to do is adding a save canvas! Let me know what do you think!

xinnanlisf commented on force-app/main/default/lwc/settingsSaveCanvas/settingsSaveCanvas.html:
```diff
@@ -23,16 +22,15 @@
-    <div class="slds-theme_default eda-height_full">
-        <div class="slds-m-left_x-small">
-            <!-- Slot that will be populated with body from parent component -->
+    <div class="slds-theme_default eda-height_full slds-m-left_x-small">
```
adding extra divs here. Also move slds-m-left_x-small to upper level.

xinnanlisf commented on force-app/main/settingsComponentLibrary/lwc/settingsRowInput/settingsRowInput.html:
```diff
@@ -1,11 +1,11 @@
 <template>
-    <div class="slds-grid slds-grid_vertical-align-start slds-wrap">
-        <div class="slds-col slds-size_2-of-3 slds-p-bottom_x-large">
+    <div class="slds-grid slds-grid_vertical-align-start slds-wrap slds-p-bottom_x-large">
```
having bottom padding for each 'setting' not each 'div' in reusable components.

xinnanlisf commented on force-app/main/settingsComponentLibrary/lwc/settingsRowDualListbox/settingsRowDualListbox.html:
```diff
@@ -1,21 +1,23 @@
-        <div class="slds-text-title slds-text-color_default slds-p-left_small slds-text-title_bold">{title}</div>
-        <div class="slds-text-body_small slds-text-color_weak slds-p-horizontal_large">{description}</div>
-    </div>
+    <div class="slds-grid slds-grid_vertical-align-start slds-wrap slds-p-bottom_x-large">
```
aligned with the other reusable component.

xinnanlisf commented:
@MatthewBlanski this should be ready i think. I added more tweaks and also reverted my changes to account page since it will be updated by Merle's PR. Let me know what you think.

MatthewBlanski reviewed (changes requested):
Please apply the styling to Account Model and Contact Information pages.

xinnanlisf commented:
@MatthewBlanski i believe the new stylings are in space for contact information and affiliations too. I am hesitating to update the account model one since we are merging naming component. It looks like we are adding some more reusable components and I do remember merging between child feature branches can be a headache sometimes. I am wondering if we can merge all these changes first and then once the account naming setting is there we can update it. I guess we can update naming branch too depends on which PR is merged first.
